include/sentry.h

```diff
@@ -717,6 +718,13 @@ SENTRY_API void sentry_options_set_debug(sentry_options_t *opts, int debug);
  */
 SENTRY_API int sentry_options_get_debug(const sentry_options_t *opts);
 
+/**
+ * Sets the sentry-native logger hook.
+ */
+SENTRY_API void sentry_options_set_loghook(sentry_options_t *opts,
```

Comment:
I wonder if we should rather name this `logger`, and explain in the comment that it is used for sentries own logging output, and how it interacts with the `debug` option. Speaking of that, I’m not sure if setting a logger should automatically override `debug`? probably not?

Reply:
renamed, comment changed